master/lib/main.dart

```diff
@@ -30,9 +29,6 @@ final ThemeData kDefaultTheme = new ThemeData(
 
 
 void main() {
-  defaultTargetPlatform == TargetPlatform.iOS
-      ? MapView.setApiKey("AIzaSyCLw1SjRi8TLDu_Nzcdo2Ufu68H1UXl9BU")
-      : MapView.setApiKey("AIzaSyDOG4RZZbkSn6Fk_OLQgayebjODUoBpLiE");
```

Comment:
!!! don't commit API Key, it is sensitive data and should not be public. 
@ankitshaw , @tanmayrauth
Please change this credentials  and use other medium to store key.

Reply:
Since the map_view is deprecated and Api Key has been exposed for too long, better if we do away with this widget and keep up with google_maps_flutter and change the credentials for map_view to prevent any misuse.

Reply:
Removing the api key is okay. I am getting it merged. It would be much appreciated if any one you look for setting up the backend logic in master directory for login feature. 
Raise another issue for the same